book/installation.rst

```diff
@@ -126,6 +126,11 @@ Read the :doc:`Symfony Release process </contributing/community/releases>`
 to better understand why there are several Symfony versions and which one
 to use for your projects.
 
+.. tip::
+
+    Note that if you install a specific version, you will want to make sure 
+    you refer to the same version of the documentation.
```

Comment:
I don't mind what the wording is, I just got hit with installing LTS (which seemed 'safest' to me) but then nothing else on the 'default' screens working (because LTS is diff enough to current).  Whatever wording you think is sufficient is fine.

I put the wording the way that it is precisely to notify people who are explicitly installing a specific version vs whatever the default version happens to be.  That's all.

Thanks.